### PR TITLE
03simplegoldenredux

### PR DESCRIPTION
**Pull request by controllhu**
Task 3 is done, please check it, thanks.

### CODE REVIEW

File: 02simplegolden/src/display.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ counter }) =>  <p>{counter}</p>;
+
+Display.propTypes = {
+  children: PropTypes.number
+  //counter: PropTypes.number.isRequired
```
Comment (gyorkibence) [inline 02simplegolden/src/display.js]: counter or children? 
pls delete the comment

File: 03simplegoldenredux/src/redux/containers/displayacorn.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+import Display from '../../components/display';
+
+const mapStateToProps = state => {
+  return {
+    clicks: state.acorn.clicks, //azért, mert combine-olva van
```
Comment (gyorkibence) [inline 03simplegoldenredux/src/redux/containers/displayacorn.js]: no comment pls

Comment (controllhu) [reply 03simplegoldenredux/src/redux/containers/displayacorn.js]: comment removed

File: 02simplegolden/src/display.js (hunk 1 above)
Comment (controllhu) [reply 02simplegolden/src/display.js]: comment removed

**Pull request merged**